Backpack-SwiftUI/Tests/AppSearchModal/BPKAppSearchModalTests.swift

```diff
@@ -54,14 +66,16 @@ class BPKAppSearchModalTests: XCTestCase {
     // MARK: - Helpers
     private func givenSut(
         with results: BPKAppSearchModalResults,
-        inputState: BPKAppSearchModal.TextFieldState = .default
+        inputState: BPKAppSearchModal.TextFieldState = .default,
+        prefixState: BPKTextField.PrefixState? = nil
```

Comment:
@yurareutskiy the test fails to compile on this line. The naming has probably changed.

Reply:
Thanks! Still not sure why it was failing before I renamed that, but amazing as it's working:)